src/pages/index.js

```diff
@@ -0,0 +1,291 @@
+import { enableValidation, settings } from "../scripts/validation.js";
+import "./index.css";
+//import initialCards from "../utils/constants.js";
+import Api from "../utils/Api.js";
+
+import logoSrc from "../images/Logo.svg";
+//import avatarSrc from "../images/avatar.jpg";
+import penSrc from "../images/pen.svg";
+import plusSrc from "../images/plus.svg";
+import penWhiteSrc from "../images/pen-white.svg";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "d11d0e4f-579c-4c1f-9935-38514d3f14dc",
+    "Content-Type": "application/json",
+  },
+});
+
+const imageLogo = document.getElementById("image-logo");
+const imageAvatar = document.getElementById("image-avatar");
+const imagePenBlack = document.getElementById("image-pen");
+const imagePenWhite = document.getElementById("image-pen-white");
+const imagePlus = document.getElementById("image-plus");
+
+imageLogo.src = logoSrc;
+imagePenBlack.src = penSrc;
+imagePenWhite.src = penWhiteSrc;
+imagePlus.src = plusSrc;
+
+const profileEditButton = document.querySelector(".profile__edit-btn");
+const editModal = document.querySelector("#edit-modal");
+
+const photoModal = document.querySelector("#photo-modal");
+const photoModalImage = photoModal.querySelector(".modal__photo");
+const photoModalCaption = photoModal.querySelector(".modal__description");
+
+const profileFormElement = document.forms["edit-profile"];
+const profileSaveBtn = profileFormElement.querySelector(".modal__save-button");
+
+const nameInput = profileFormElement.querySelector("#name");
+const jobInput = profileFormElement.querySelector("#description");
+
+const profileNameElement = document.querySelector(".profile__name");
+const profileJobElement = document.querySelector(".profile__description");
+
+api
+  .getAppInfo()
+  .then((result) => {
+    const savedCards = result[0];
+    const savedProfile = result[1];
+    imageAvatar.src = savedProfile["avatar"];
+    imageAvatar.alt = savedProfile["name"];
+    profileNameElement.textContent = savedProfile["name"];
+    profileJobElement.textContent = savedProfile["about"];
+    savedCards.forEach((card) => {
+      const cardEl = getCardElements(card);
+      cardsList.append(cardEl);
+    });
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+function openModal(modal) {
+  modal.classList.add("modal_opened");
+  document.addEventListener("keydown", escapeListener);
+}
+
+function closeModal(modal) {
+  modal.classList.remove("modal_opened");
+  document.removeEventListener("keydown", escapeListener);
+}
+
+function escapeListener(evt) {
+  const modal = document.querySelector(".modal_opened");
+  if (evt.key === "Escape") {
+    closeModal(modal);
+  }
+}
+
+const modalList = document.querySelectorAll(".modal");
+modalList.forEach((modal) => {
+  modal.addEventListener("click", function (evt) {
+    if (evt.target.parentElement.classList.contains("page")) {
```

Comment:
`parentNode` or `parentElement` should not be used in the project because the order can be changed (or someone can add other markup) in the future and your code will not work anymore. Method `closest` can help you here


The overlay can be easily checked like this

```js

    if (evt.target.classList.contains("modal")) {
      closeModal(modal);
    }
```